scripts/docker-stop.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -xeuo pipefail
+
+# the Jekyll version used by GitHub is given at https://pages.github.com/versions/,
```

Comment:
We don't use the versions at that url, have a look at the Gemfile and lock file to see which we use

Reply:
removed this outdated comment, this was for the `jekyll/jekyll` image